apps/full-site-editing/full-site-editing-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -161,13 +161,16 @@ public function enqueue_assets() {
 	public function fetch_vertical_data() {
 		$vertical_id        = get_option( 'site_vertical', 'default' );
 		$transient_key      = implode( '_', [ 'starter_page_templates', $vertical_id, get_locale() ] );
-		$vertical_templates = get_transient( $transient_key );
+		$vertical_templates = apply_filters( 'fse_spt_vertical_templates_from_cache', get_transient( $transient_key ) );
```

Comment:
Instead of adding our own filter, we could use the existing one in get_transient: `transient_$transient_key`.

If you feel like regenerating the transient key is too much work, we could also check for `WP_DEBUG` being defined and true, after `false === $vertical_templates`. Which is probably the best development solution long term.

Reply:
TIL! 

Ok I've gone with `WP_DEBUG`.